setup.py

```diff
@@ -24,7 +27,7 @@
     keywords='SVG, PDF, reportlab, conversion, graphics',
     url='https://github.com/deeplook/svglib',
     install_requires=install_requires,
-    setup_requires=['pytest-runner'],
+    setup_requires=[] + pytest_runner,
```

Comment:
left the [] in to signal a place where traditional setup_requires libs could be added - but it's also entirely redundant here